sdks/wasm/Makefile

```diff
@@ -118,3 +110,5 @@ run-%: toolchain build
 clean:
 	
 package:
+
+FORCE:
```

Comment:
what's that target for?

Reply:
This is to make the targets that depend on it to be always reevaluated. IE, make them behave just like PHONY target.

The bcl test suite targets must be PHONY so we get proper dep tracking by the nested make invocation.
This makes the dev cycle nicer as I can edit the test suite and just type `make run-foo` and not have to clear or touch files.

The alternative to this trick is to rename those targets from actual files to proper PHONY targets. Which I find unfortunate, as the actual file being generated gets obscured.

Reply:
Why don't you do `.PHONY: $(TOP)/mcs/class/lib/wasm/tests/wasm_corlib_test.dll`? That will force to reevaluate the `$(TOP)/mcs/class/lib/wasm/tests/wasm_corlib_test.dll` target, even if the file exists